roles/sosreport/tasks/main.yml

```diff
@@ -36,33 +39,47 @@
   ansible.builtin.command: "sosreport --batch --tmp-dir {{ sosreport_tmpdir }} -z gzip --name {{ inventory_hostname_short }} -o {{ sosreport_plugins|join(',') }}"
   changed_when: true
 
-- name: Rename md5 file
-  ansible.builtin.command: "mv -f {{ sosreport_tmpdir }}/*.md5 {{ sosreport_tmpdir }}/{{ sosreport_archive_filename }}.md5"
+- name: Rename sha256 file
+  become: true
+  ansible.builtin.shell: "mv -f {{ sosreport_tmpdir }}/*.sha256 {{ sosreport_tmpdir }}/{{ sosreport_archive_filename }}.sha256"
   changed_when: true
 
 - name: Rename archive
-  ansible.builtin.command: "mv -f {{ sosreport_tmpdir }}/*.tar.gz {{ sosreport_tmpdir }}/{{ sosreport_archive_filename }}"
+  become: true
+  ansible.builtin.shell: "mv -f {{ sosreport_tmpdir }}/*.tar.gz {{ sosreport_tmpdir }}/{{ sosreport_archive_filename }}"
   changed_when: true
 
-- name: Fetch md5 file
+- name: Change report mode
   become: true
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ operator_user }}"
+    group: "{{ operator_group }}"
+    mode: 0755
+    recurse: true
+  loop:
+    - "{{ sosreport_tmpdir }}"
```

Comment:
The loop here makes no sense. I think you can enter this path directly here at Path.

Reply:
You are right. It makes no sense. I changed it.